tests/components/nest/device_info_test.py

```diff
@@ -0,0 +1,103 @@
+"""Test for properties for devices common to all entity types."""
+
+from google_nest_sdm.device import Device
+
+from homeassistant.components.nest.device_info import DeviceInfo
+
+
+def test_device_custom_name():
+    """Test a device name from an Info trait."""
+    device = Device.MakeDevice(
+        {
+            "name": "some-device-id",
+            "type": "sdm.devices.types.DOORBELL",
+            "traits": {
+                "sdm.devices.traits.Info": {
+                    "customName": "My Doorbell",
+                },
+            },
+        },
+        auth=None,
+    )
+
+    device_info = DeviceInfo(device)
+    assert device_info.device_name == "My Doorbell"
+    assert device_info.device_model == "Doorbell"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)